receiver/apachesparkreceiver/metadata.yaml

```diff
@@ -4,4 +4,689 @@ status:
+  spark.driver.code_generator.source_code.average_size:
+    description: Average size of the source code generated by a CodeGenerator code generation operation.
+    enabled: true
+    gauge:
+      value_type: double
+    unit: bytes
+    attributes: []
+  spark.driver.dag_scheduler.jobs.active:
+    description: Number of active jobs currently being processed by the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ jobs }"
+    attributes: []
+  spark.driver.dag_scheduler.jobs.all:
+    description: Number of jobs that have been submitted to the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ jobs }"
+    attributes: []
+  spark.driver.dag_scheduler.stages.failed:
+    description: Number of failed stages run by the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ stages }"
+    attributes: []
+  spark.driver.dag_scheduler.stages.running:
+    description: Number of stages the DAGScheduler is currently running.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ stages }"
+    attributes: []
+  spark.driver.dag_scheduler.stages.waiting:
+    description: Number of stages waiting to be run by the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ stages }"
+    attributes: []
+  spark.driver.live_listener_bus.events_posted:
+    description: Number of events that have been posted on the LiveListenerBus.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ events }"
+    attributes: []
+  spark.driver.live_listener_bus.listener_processing_time.average:
+    description: Average time taken for the LiveListenerBus to process an event posted to it.
+    enabled: true
+    gauge:
+      value_type: double
+    unit: ms
+    attributes: []
+  spark.driver.live_listener_bus.events_dropped:
+    description: Number of events that have been dropped by the LiveListenerBus.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ events }"
+    attributes: []
+  spark.driver.live_listener_bus.queue_size:
+    description: Number of events currently waiting to be processed by the LiveListenerBus.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ events }"
+    attributes: []
+  spark.driver.jvm_cpu_time:
+    description: Current CPU time taken by the Spark driver.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: ns
+    attributes: []
+  spark.driver.executor_metrics.jvm_memory:
```

Comment:
Is "executor metrics" a component in spark? Sounds more like a category of metrics.

Reply:
It is a category of metrics.